recipes/czifile/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "czifile" %}
+{% set version = "2019.7.2" %}
+{% set sha256 = "04c0e6bed3b24d1bf42bc2cf899a5a08986641379305ce88600fd1c710486436" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  noarch: python
+  entry_points:
+    - czifile = czifile.czifile:main
+    - czi2tif = czifile.czi2tif:main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - tifffile
+    - imagecodecs-lite
+
+test:
+  imports:
+    - czifile
+  commands:
+    - czi2tif
```

Comment:
I think we should also be testing the `czifile` command

Reply:
The problem is, that the `czifile` command itself expects a filename as argument, or will open a file picker… ( https://gist.github.com/csachs/5388c2a8b406f351e3be9b5782f0f5e4#file-czifile-py-L1296..L1304 ), and will try to open whatever it got passed.
Hence I didn't add it for a check. Both commands are not testing much more than the import check, anyways.
As a workaround, due to the `if`, starting it with an empty argument `czifile ''` does indeed run it without any errors.